src/index.ts

```diff
@@ -46,8 +46,8 @@ export function map<T>(opt_initial: T | undefined) {
  * @returns {boolean}
  * @template T
  */
-export function hasOwn<T>(obj: T, key: string) {
-  return _hasOwn.call(obj, key);
+export function hasOwn<T>(object: T, key: string) {
```

Comment:
see https://mdn.io/reflect-has

Reply:
> see https://mdn.io/reflect-has

I think this function is closer to https://mdn.io/object-hasown (which was standardized _after_ we published this package). The difference is this function distinguishes whether the property is owned (not inherited).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn